app/assets/javascripts/select-kit/addon/components/user-chooser.js

```diff
@@ -27,6 +27,7 @@ export default MultiSelectComponent.extend({
     allowEmails: false,
     groupMembersOf: undefined,
     excludeCurrentUser: false,
+    customOptions: {},
```

Comment:
What do you think of naming it customSearchOptions instead?

Reply:
Doesn't this mean every instance of the component gets the same object instance? That's probably not what we want.